holoviews/plotting/plot.py

```diff
@@ -527,7 +532,7 @@ def init_comm(self):
         """
         comm = None
         if self.dynamic or self.renderer.widget_mode == 'live':
-            comm = self.renderer.comms[self.renderer.mode][0](self)
```

Comment:
I assume it was parameterized by ``mode`` was because of nbagg support? Does that still work?

Reply:
Ah, forgot about that, I was going to suggest deprecating it, I doubt it works right now.

Reply:
I'm fully in favor of deprecating nbagg myself. @jbednar ?

Reply:
I've now removed nbagg.